grakn-client/src/main/java/ai/grakn/client/BatchExecutorClient.java

```diff
@@ -120,19 +121,19 @@ private BatchExecutorClient(Builder builder) {
     /**
      * Will block until there is space for the query to be submitted
      */
-    public Observable<QueryResponse> add(Query<?> query, Keyspace keyspace) {
+    public Observable add(Query<?> query, Keyspace keyspace) {
```

Comment:
You missed a spot with your raw type fixing! This one is important because it's the entry-point to using the `BatchExecutorClient`.

If we don't want to expose the `QueryResponse` type (since it is useless), we can make this return `Observable<?>`.